test/bridge.jl

```diff
@@ -162,6 +162,7 @@ MOIU.@model(NoRSOCModel,
 
     @testset "Combining two briges" begin
         fullbridgedmock = MOIB.fullbridgeoptimizer(mock, Float64)
+        @test MOI.supports_constraint(fullbridgedmock, MOI.VectorAffineFunction{Float64}, MOI.PositiveSemidefiniteConeSquare)
```

Comment:
This test doesn't fall under the "combining two bridges" header. Consider using a different testset.